crates/sc-consensus-subspace/src/notification.rs

```diff
@@ -29,7 +29,7 @@ type SharedNotificationSenders<T> = Arc<Mutex<Vec<TracingUnboundedSender<T>>>>;
 
 /// The sending half of the Subspace notification channel(s).
 #[derive(Clone)]
-pub(crate) struct SubspaceNotificationSender<T: Clone + Send + Sync + fmt::Debug + 'static> {
+pub struct SubspaceNotificationSender<T: Clone + Send + Sync + fmt::Debug + 'static> {
```

Comment:
Why is this and `notify` method are public?

Reply:
Is there a way to fire an event otherwise? I needed to fire an event from other crates and made this method public.

Reply:
If you need to fire an event from other places, you probably should use a different API and not this one. This one was designed to be private.